### PR TITLE
upload artifact

### PR DESCRIPTION
**Pull request by dannypsnl**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: .github/workflows/witc-release.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    name: Create Github Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: haskell/actions/setup@v2
+        with:
+          ghc-version: "9.2.8"
+      - run: stack install
+
+      - name: Upload PDF file
```
Comment (dm4) [inline .github/workflows/witc-release.yml:39]: Why PDF file?

Comment (dannypsnl) [reply .github/workflows/witc-release.yml:39]: oops

**Review by dm4 — APPROVED** [review]
👍

**Pull request merged**